### PR TITLE
Align column filter dropdown with icon

### PR DESCRIPTION
**Pull request by mayank99**
Changed placement from `bottom` to `bottom-start`.

| Before | After |
| --- | --- |
| image | image |

## Checklist

- [ ] Add meaningful unit tests for your component (verify that all lines are covered)
- [x] Verify that all existing tests pass
- [x] ~~Add component features demo in Storybook (different stories)~~
- [x] Approve test images for new stories
- [ ] Add an entry to the changelog for any new components and changes
- [x] Add screenshots of the key elements of the component
- [x] Add any additional comments describing the features you've implemented

### CODE REVIEW

File: stories/core/Table.stories.tsx [hunk 1]
```diff
@@ -486,6 +486,26 @@ Filters.args = {
...
+Filters.parameters = {
+  creevey: {
+    tests: {
+      async open() {
+        const nameFilter = (
+          await this.browser.findElements({
+            css: '.iui-filter-button',
+          })
+        )[1];
+        await this.browser.actions().move({ origin: nameFilter }).perform();
```
Comment (veekeys) [inline stories/core/Table.stories.tsx:498]: you need to move there before clicking?

Comment (mayank99) [reply stories/core/Table.stories.tsx:498]: Yeah it didn't work without

Comment (veekeys) [reply stories/core/Table.stories.tsx:498]: maybe stack click on the same actions then? `...move({origin: nameFilter}).click().perform()`

Comment (mayank99) [reply stories/core/Table.stories.tsx:498]: I could, but that reaches the line limit, so it gets wrapped into 4 lines

**Review by veekeys — APPROVED** [review]

**Pull request merged**